--- cppunit-1.10.2/cppunit.m4.orig	2002-08-22 15:20:59.000000000 +0200
+++ cppunit-1.10.2/cppunit.m4	2004-07-18 01:42:22.019016480 +0200
@@ -1,7 +1,7 @@
 dnl
 dnl AM_PATH_CPPUNIT(MINIMUM-VERSION, [ACTION-IF-FOUND [, ACTION-IF-NOT-FOUND]])
 dnl
-AC_DEFUN(AM_PATH_CPPUNIT,
+AC_DEFUN([AM_PATH_CPPUNIT],
 [
 
 AC_ARG_WITH(cppunit-prefix,[  --with-cppunit-prefix=PFX   Prefix where CppUnit is installed (optional)],
